skymp5-server/cpp/server_guest_lib/WorldState.cpp

```diff
@@ -234,6 +234,9 @@ const std::shared_ptr<MpForm>& WorldState::LookupFormById(uint32_t formId)
 {
   static const std::shared_ptr<MpForm> kNullForm;
   auto it = forms.find(formId);
+  // It may loop back to the beginning otherwise.
+  if (it == forms.end() && it != forms.begin())
```

Comment:
???
don't understand

Reply:
Sorry for the confusion. It's probably okay to remove.
When I looked it up.. I misunderstood the definition of the find function.